backend/django_core/templates/tailwind_base.html

```diff
@@ -28,6 +28,7 @@
         />
         <script src="{% static 'vendor/js/htmx/htmx.min.js'%}"></script>
         <script src="{% static 'vendor/js/hyperscript/hyperscript.min.js'%}"></script>
+        <script src="{% static 'vendor/js/swiper/swiper.min.js'%}"></script>
```

Comment:
Importing something like swiper at this level is not a good practice

Reply:
> Importing something like swiper at this level is not a good practice

Then? import inside `_partial.html`?

Reply:
> Then? import inside _partial.html?

Correct, import the function where you declare swiper class